### ISSUE TITLE
 lower batch size ：	2 labeled + 2 unlabeled -----> bad AP

### ISSUE THREAD

**Issue by Saixiaoma** [facebookresearch/unbiased-teacher#53]
Thanks for the great work.
I have been training the code with your instruction in README.

Due to device limitations, there is only one GPU and only 12g of memory. In order to train, I can only adjust the batchsize to 2, which leads to very poor final results, with an AP of only about 11%. 

I would like to ask whether batchsize has such a big impact on the result.

I'm looking forward to your reply !!

**Comment by hachreak** [comment]
Did you try to scale the learning rate?

**Comment by Saixiaoma** [comment]
Yes. I conducted two more experiments afterwards. One was to repeat the experiment without modifying any parameters, and the other was to modify the learning rate 0.01 to 0.0025. The results of both times were much worse than the initial one.
